src/Command/ProcessCommand.php

```diff
@@ -145,8 +142,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $variantCollection = $variantConfiguration->getVariantCollection();
         Assert::notNull($variantCollection);
 
-        $this->eventDispatcher->dispatch(new ProcessingStartedEvent($variantCollection));
```

Comment:
No reason to remove this. Other plugin users could just as well want to hook into the lifecycle of the processing by listening to this event

Reply:
True, good point.